### PR TITLE
Chore: speeding up doc building

### PR DESCRIPTION
**Pull request by ziyoung**
Please make sure these boxes are checked before submitting your PR, thank you!

* [ ] Make sure you follow Element's contributing guide ([中文](https://github.com/ElemeFE/element/blob/master/.github/CONTRIBUTING.zh-CN.md) | [English](https://github.com/ElemeFE/element/blob/master/.github/CONTRIBUTING.en-US.md) | [Español](https://github.com/ElemeFE/element/blob/master/.github/CONTRIBUTING.es.md) | [Français](https://github.com/ElemeFE/element/blob/master/.github/CONTRIBUTING.fr-FR.md)).
* [ ] Make sure you are merging your commits to `dev` branch.
* [ ] Add some descriptions and refer relative issues for you PR.

重写 markdown-loader。`npm run dev` 时间由 **264.017s** 变成 **97.509s**🚀。

### CODE REVIEW

**Comment by ziyoung** [comment]
把文档迁移一下（删除多余的 script）那么基本上功能就完成了。

**Comment by ziyoung** [comment]
在升级 webpack 的 pr 中，把 external 给去掉了。https://github.com/ElemeFE/element/pull/14173/files#diff-47f0960b0efebeee762312ff2c551411
可以再调整一下。

**Comment by ziyoung** [comment]
2.5.x 多语言文档 js 大小
```
2.4M en-US.0ee9075.js
2.4M es.0ee9075.js
2.5M zh-CN.0ee9075.js
```

2.6.x 多语言文档 js 大小
```
1.0M en-US.98246ce.js
1.1M es.98246ce.js
1.1M fr-FR.98246ce.js
1.1M zh-CN.98246ce.js
```

File: build/md-loader/config.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+const Config = require('markdown-it-chain');
```
Comment (island205) [inline build/md-loader/config.js]: md-loader 为啥 内置？

**Pull request merged**